content/realtime/authentication.textile

```diff
@@ -117,15 +121,27 @@ Below is a rather contrived yet straightforward example that instances a Realtim
 
 h4(#token-upgrading). Upgrading the token on a connection
 
-The token that a connection uses can be upgraded whilst remaining connected. When a token is nearly expired, Ably will prompt the library to request a new token (using the <span lang="ruby,python">@auth_url@ or @auth_callback@</span><span lang="default">@authUrl@ or @authCallback@</span>). Once the library has obtained a new token, it will upgrade the connection to use the new token. This all happens automatically and seamlessly.
+The token that a connection uses can be upgraded whilst remaining connected. When a token is nearly expired, Ably will prompt the library to request a new token (using the <span lang="ruby,python">@auth_url@ or @auth_callback@</span><span lang="default">@authUrl@ or @authCallback@</span>). Once the library has obtained a new Ably-compatible token, it will upgrade the connection to use the new token. This all happens automatically and seamlessly.
```

Comment:
I think this (final sentence) is example where simply saying "token" is quite clear, and "Ably-compatible" seems to be suggesting that there's something special about the token it's referring to whereas (in this context, where everything is Ably-compatible) there isn't.